lib/pages/view-post-page.js

```diff
@@ -68,7 +68,7 @@ export default class ViewPostPage extends BaseContainer {
 
 	commentEventuallyShown( comment ) {
 		const self = this;
-		const commentSelector = By.xpath( `//p[text() = "${comment}"]` );
+		const commentSelector = By.css( `p:contains("${comment}")` );
```

Comment:
I couldn't get this selector to work in the Chrome console. Is this a deprecated selector?